src/dymoprint/lib/utils.py

```diff
@@ -33,12 +33,16 @@ def draw_image(bitmap):
         del drawobj
 
 
-def px_to_mm(px):
+def px_to_mm(px) -> float:
     mm = px / PIXELS_PER_MM
     # Round up to nearest 0.1mm
     return math.ceil(mm * 10) / 10
 
 
+def mm_to_px(mm) -> float:
```

Comment:
```suggestion
def mm_to_px(mm) -> int:
```

Reply:
I prefer to have pixel as float still. It indeed makes no physical sense, but helps with calculations along the way. I prefer to convert to int at the very end.

Reply:
Added one more commit. Please review.